lib/katello/tasks/receptor/extract_orgs.rake

```diff
@@ -0,0 +1,27 @@
+namespace :katello do
+  namespace :receptor do
```

Comment:
Was just perusing and got curious if the receptor namespace here was required given its a generic-ish rake task.

Reply:
I wanted to namespace this in way that might make a user second guess using this task - which they would not have a reason to. Maybe a printed warning would help in clarifying that the result / output of the task has no guarantees other than to serve the needs of the receptor plugin